packages/server/lib/controllers/files.js

```diff
@@ -39,13 +39,14 @@ module.exports = {
     })
   },
 
-  handleCrossOriginIframe (req, res) {
+  handleCrossOriginIframe (req, res, namespace) {
```

Comment:
These are changes to make cypress in cypress work better (not totally fixed yet)